spanner_orm/tests/model_test.py

```diff
@@ -19,12 +19,130 @@
 from unittest import mock
 
 from absl.testing import parameterized
+from spanner_orm import error
 from spanner_orm import field
 from spanner_orm.tests import models
 
 
 class ModelTest(parameterized.TestCase):
 
+  @mock.patch('spanner_orm.table_apis.find')
+  def test_find_calls_api(self, find):
+    mock_transaction = mock.Mock()
+    models.UnittestModel.find(
+        mock_transaction, string='string', int_=1, float_=2.3)
+
+    find.assert_called_once()
+    (transaction, table, columns, keyset), _ = find.call_args
+    self.assertEqual(transaction, mock_transaction)
+    self.assertEqual(table, models.UnittestModel.table)
+    self.assertEqual(columns, models.UnittestModel.columns)
+    self.assertEqual(keyset.keys, [[1, 2.3, 'string']])
+
+  @mock.patch('spanner_orm.table_apis.find')
+  def test_find_result(self, find):
+    mock_transaction = mock.Mock()
+
+    find.return_value = [['key', 'value_1', None]]
+    result = models.SmallTestModel.find(mock_transaction, key='key')
+    if result:
+      self.assertEqual(result.key, 'key')
+      self.assertEqual(result.value_1, 'value_1')
+      self.assertIsNone(result.value_2)
+    else:
+      self.fail('Failed to find result')
```

Comment:
Just a question, no need to change anything. Do you see an advantage to doing this instead of 
```
self.assertIsNotNone(result);
```

(Also not sure it it matters in this codebase, but I think it should be `if result is not None`.

Reply:
I agree with both points. There's a lot of cleanup that could be done.

Reply:
Same as above.